1-Draft/RFC0000-RFC-Enhance-some-cmdlets-with-Culture-and-Ordinal.md

```diff
@@ -0,0 +1,55 @@
+---
+RFC: RFC0000
+Author: Ilya Sazonov
+Status: Draft
+Area: Commands
+Comments Due: 11/10/2019
+---
+
+# Introduction
+
+Historically, PowerShell has only used linguistic (`CurrentCulture`) and case-insensitive in many places to make string comparisons.
+Goal of the RFC is to add more flexibility and performance in cmdlets based on string comparison operations.
+
+## Motivation
+
+In some scenarios PowerShell users need to do string comparisons in different culture than current culture.
+
+Also sometimes PowerShell users need to do non-linguistic, binary/ordinal, comparisons.
+
+Such scenarios would be communicating with localized applications and searching in log files.
+
+Users want to get performance comparable to utilites like `grep`, `egrep` and `ripgrep`.
+In the RFC context it is achieved by `Ordinal` and `OrdinalIgnoreCase` C# string comparison.
+
+## Specification
+
+### Affected cmdlets
+
+- `Compare-Object`
+- `Group-Object`
+- `Sort-Object`
+- `Select-Object`
+- `Select-String`
+
+### Proposal
+
+#### `Culture` parameter
+
+Add `Culture` parameter to all affected cmdlets if absent.
+
+`Culture` parameter accepts C# `CultureInfo` type, `Ordinal`, `Invariant` and `Current` values.
+It is default parameter set for backward compatibility and default value is `Current` (C# `CurrentCulture`).
+
+##### Samples
+
+```powershell
+Select-String -Culture ru-RU -CaseSensitive      # lingustic as expected for Culture term
```

Comment:
Can you add some of the examples you already have in the code PR tests as examples here?  Specifically looking to have both inputs and expected outputs.

Reply:
Done.